package.json

```diff
@@ -492,9 +492,35 @@
           "command"
         ],
         "properties": {
+          "label": {
+            "type": "string",
+            "description": "The name of the task"
+          },
           "command": {
             "type": "string",
-            "description": "CMake command"
+            "description": "CMake build command"
+          },
+          "target": {
```

Comment:
Let's save this for later.  I think we want to do "targets" as an array of string instead anyway.  You also mentioned support for multiple targets in the status bar that we don't have.  We should have issues for both.

The target we select for this task will be the active target (or "all" if no target selected)